docs/auto_arpa.md

```diff
@@ -0,0 +1,91 @@
+## Automatic PTR Generation With auto_arpa
+
+octoDNS supports the automatic generation of `PTR` records for in-addr.arpa. and ip6.arpa. zones. In order to enable the functionality the `auto_arpa` key needs to be passed to the manager configuration.
+
+```yaml
+---
+manager:
+  auto_arpa: true
+```
+
+Alternatively the value can be a dictionary with configuration options for the AutoArpa processor/provider.
+
+```yaml
+---
+manager:
+  auto_arpa:
+    ttl: 1800
+```
+
+Once enabled a singleton `AutoArpa` instance, `auto-arpa`, will be added to the pool of providers and globally configured to run as the very last global processor so that it will see all records as they will be seen by targets. Further all zones ending with `arpa.` will be held back and processed after all other zones have been completed so that all `A` and `AAAA` records will have been seen prior to planning the `arpa.` zones. 
+
+In order to add `PTR` records for a zone the `auto-arpa` source should be added to the list of sources for the zone.
+
+```yaml
+0.0.10.in-addr.arpa.:
+  sources:
+    - auto-arpa
+  targets:
+    - ...
+```
+
+The above will add `PTR` records for any `A` records previously seen with IP addresses 10.0.0.*.
+
+### A Complete Example
+
+#### config/octodns.yaml
+
+```yaml
+manager:
+  auto_arpa: true
+
+providers:
+  config:
+    class: octodns.provider.yaml.YamlProvider
+    directory: tests/config
+
+  powerdns:
+    class: octodns_powerdns.PowerDnsProvider
+    host: 10.0.0.53
```

Comment:
I consider it best practice to refer to it by IP rather than name since you could push something to the server that breaks the name and then not be able to fix anything. That should probably be spelled out in the octodns-powerdns docs if it's not already. I can put a non-matching IP here to make it more distinct/unrelated.

Reply:
Yeah I didn't IP would help, I'd just just another provider to avoid mentioning any IP altogether, assuming powerdns provider is just a sample that's irrelevant to delayed arpa.